docs/resources/slo.md

```diff
@@ -503,6 +504,17 @@ Optional:
 - `uql` (String) UQL query
 
 
+<a id="nestedblock--objective--count_metrics--total--logic_monitor"></a>
+### Nested Schema for `objective.count_metrics.total.logic_monitor`
+
+Required:
+
+- `device_data_source_instance_id` (Number) Device Datasource Instance ID
+- `graph_id` (Number) Graph ID
+- `line` (String) Line
+- `query_type` (String) Query type: device_metrics
+
+
```

Comment:
this comment applies to all markdown files, it seems you edited these manually, which is no the way to go :)

You need to do the following things:

- add example to `examples` folder
- add documentation to docs templates located under `templates`

Reply:
No, it was generated by command: `make generate`